buildtest/menu/__init__.py

```diff
@@ -183,6 +184,17 @@ def show_menu(self):
             "-v", "--version", help="choose a specific version of schema to show.",
         )
         parser_schema.add_argument(
-            "-n", "--name", help="show schema by name (e.g., script)",
+            "-n",
+            "--name",
+            help="show schema by name (e.g., script)",
+            choices=supported_schemas,
+        )
+        # can't use attribute args.global so setting dest=main for argparse
```

Comment:
Maybe just do args._global instead? It would still be intuitive , remove the need for this comment, and not trigger the error.